app/Keycloak/Service/ApiClientInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Keycloak\Service;
+
+use App\Domain\Integrations\Integration;
+use App\Keycloak\Client;
+use App\Keycloak\Realm;
+use Ramsey\Uuid\UuidInterface;
+
+interface ApiClientInterface
```

Comment:
I am not a fan of using the `interface` suffix. I think we completely avoided this in this repository.